UPGRADE.md

```diff
@@ -58,6 +58,10 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 - *(optional)* [#596 Trusted proxies are now configurable in parameters.yml file](https://github.com/shopsys/shopsys/pull/596)
     - for easier deployment to production, make the trusted proxies in `Shopsys\Boostrap` class loaded from DIC parameter `trusted_proxies` instead of being hard-coded
 
+### [shopsys/framework]
+- [#627 model service layer removal](https://github.com/shopsys/shopsys/pull/627)
+    - please read upgrade instruction in [separated article](docs/upgrade/services_removal.md)
```

Comment:
I would use "separate" as this is the way we use it :) however, I do not really care, it seems that both terms are acceptable https://english.stackexchange.com/questions/13144/separated-versus-separate